suspect/io/twix.py

```diff
@@ -382,44 +382,53 @@ def load_twix(filename):
 
 def anonymize_twix_header(header_string):
     """Removes the PHI from the supplied twix header and returns the sanitized version.
-
     This consists of:
     1) Replacing the patient id and name with strings of lower case x
     characters.
-    2) Replacing the patient birthday with 19700101
```

Comment:
No, 19900101 is what we've been using at our site so I wanted to keep it consistent but this doesn't matter.